### DISCUSSION TITLE
建议新增：可选的随机端口号及密钥

### DISCUSSION THREAD

**Discussion by Tseshongfeeshur** [MemDeco-WG/RootModuleMaker#62] [Ideas 想法 & 功能提议]
为了确保安全，鄙人建议在模块安装时允许用户选择不同于默认的端口号（包括代理和 api ）及 api 访问密钥。

**Comment by LIghtJUNction** [comment]
> 为了确保安全，鄙人建议在模块安装时允许用户选择不同于默认的端口号（包括代理和 api ）及 api 访问密钥。

端口号这个完全可以安装后自己修改，在模块安装过程修改感觉没必要-api访问密钥，你是指的管理面板的密码吗？这个在配置文件里有，默认为空，也是用户自己决定要不要修改，我可以提醒一下
